platform-sdk/sdk/settings.txt

```diff
@@ -21,7 +21,7 @@ sync.syncProtocolPermitCount,             2
                                           #############################
 
 freezeSecondsAfterStartup,                0
-event.creation.maxCreationRate,           5
+event.creation.maxCreationRate,           20

```

Comment:
Was this change intentional?  If so, might want to have Austin or Cody weigh in on its value.

Reply:
Not intentional. Thanks @edward-swirldslabs